package/src/monaco.contribution.ts

```diff
@@ -194,6 +194,8 @@ monaco.editor.onDidCreateEditor((editor) => {
     }
 
     triggerSuggestDialogWhenCompletionItemSelected(editor);
+
+    // editor.updateOptions({ suggest: { filterGraceful: false } });
```

Comment:
Left in by accident?

Reply:
No. I know I will need to use it for something else in my next feature, so I left it. 
Today it will be gone. 
I promise :)